packages/shared/components/FormFieldWrapper.tsx

```diff
@@ -49,7 +49,7 @@ const FormFieldWrapper = ({
       : "";
   };
 
-  const getHintContent = hintContent => {
+  function HintContent({ hintContent }) {
```

Comment:
This should be a constant called HintContent to follow the pattern of other components in our application.
`const HintContent = ({hintContent}) => {...}`

Will need to define the prop type for hintContent. 

Then below the render props will likely need to be refactored so that it can take in actual components. I'm not sure all that will entail without digging into it.